plugins/GSdx/GSCapture.cpp

```diff
@@ -505,7 +518,7 @@ bool GSCapture::BeginCapture(float fps, GSVector2i recommendedResolution, float
 
 	m_capturing = true;
 
-	return true;
+	return new std::wstring(fn);
```

Comment:
This seems to only be defined in the `_WIN32` ifdef so you're getting linux compiler errors - https://travis-ci.org/github/PCSX2/pcsx2/jobs/723997550#L1077

Reply:
ah right

Reply:
Only one problem, I have no idea what the linux pieces are doing.